docs/source/changelog.rst

```diff
@@ -37,7 +37,7 @@ Array
 
 DataFrame
 +++++++++
-
+- Bugfix to allow column assignment of pandas datetimes
```

Comment:
Can I ask you to move this bullet point up to the new and blank sections above?  (we've just had a release).  Also, if you feel like adding the PR number and your name as modeled in the other lines that would be convenient.  You can match your name to a personal website down at the bottom of this page.

Reply:
Will do now. Thanks again for the guidance.